qiskit_nature/second_q/algorithms/excited_states_solvers/qeom.py

```diff
@@ -268,6 +270,7 @@ def solve(
         Args:
             problem: A class encoding a problem to be solved.
             aux_operators: Additional auxiliary operators to evaluate.
+
         Returns:
             An interpreted :class:`~.EigenstateResult`. For more information see also
             :meth:`~.BaseProblem.interpret`.
```

Comment:
Now QEOMResult, it seems, does not appear in the docs. I think the only place where we expose this to an end user is via the raw_result field of the EigenstateResult that is returned here from this solve method. So I thinking that we can expand this returns text to state that fact and link the QEOMResult class here so that it can be navigated to from here directly in the context where its relevant, rather than try to put it elsewhere like other classes where we have them documented and exposed via the init.